src/components/Pages/reserve/Reserve.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './style.css';
+import { useForm } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { createReservation } from '../../../redux/reservations/reserve';
+
```

Comment:
Creating a file `.jsx` is an old way to create components. Microverse teach us a new way to create components and how to use them. It makes our code uniform. It doesn't make the code looks good if some components are created with `.jsx` and some of them are created with `.js`. Kindly create only `.js` files. 😃

Reply:
okay i will fix it this parts